bitcoin/src/blockdata/script/mod.rs

```diff
@@ -678,66 +678,19 @@ pub enum Error {
     EarlyEndOfScript,
     /// Tried to read an array off the stack as a number when it was more than 4 bytes.
     NumericOverflow,
-    /// Error validating the script with bitcoinconsensus library.
-    #[cfg(feature = "bitcoinconsensus")]
-    BitcoinConsensus(bitcoinconsensus::Error),
     /// Can not find the spent output.
     UnknownSpentOutput(OutPoint),
```

Comment:
In 164479fb3589f501f2778929b2cbc3621bd782a5:

I guess you want to delete `UnknownSpentOutput` as well.

Reply:
ooo, good reviewing. Done.